### ISSUE TITLE
Refreshing the documentation ?

### ISSUE THREAD

**Issue by NikosAlexandris** [fsspec/kerchunk#410]
The current [Kerchunk documentation](https://fsspec.github.io/kerchunk/index.html) is minimal in a good way and the content to the point.  Would you, however, see value added to refresh its look, feel and interactivity?  Both https://squidfunk.github.io/mkdocs-material/ and https://jupyterbook.org/en/stable/intro.html are awesome tools to improve the user interface and experience of a project's documentation.  @martindurant, any interest ?

**Comment by martindurant** [comment]
You are absolutely right, we could use a lot more and better quality documentation - the stuff we never get to while trying to chase features. The problem is more of getting the content out rather than the tools to make it into pages I think, but I'd be happy to consider any tool that helps. We do have some more book-like content as a pythia cookbook: https://projectpythia.org/kerchunk-cookbook/README.html

**Comment by NikosAlexandris** [comment]
> You are absolutely right, we could use a lot more and better quality documentation - the stuff we never get to while trying to chase features. The problem is more of getting the content out rather than the tools to make it into pages I think, but I'd be happy to consider any tool that helps. We do have some more book-like content as a pythia cookbook: https://projectpythia.org/kerchunk-cookbook/README.html

This is based on Jupyter Book ?  I have quite some experience using it.

Also, I guess it is known, yet another mention doesn't hurt!  Good way(s) on doing documentation from https://diataxis.fr/:

> The Diátaxis framework aims to solve the problem of structure in technical documentation. It adopts a systematic approach to understanding the needs of documentation users in their cycle of interaction with a product.

The principles laid out there have helped me a lot as I am working out a somewhat complex project at work.

**Comment by NikosAlexandris** [comment]
@martindurant Can I suggest some of the collected content I have over at https://nikosalexandris.github.io/rekx/ for integration in the Kerchunk documentation ? Any other plans for the docs (switching to MkDocs or JupyterBook ?

**Comment by martindurant** [comment]
Please do add any and all documentation you have! The format is not too important to me.
